api/standard/resource.go

```diff
@@ -92,6 +92,34 @@ const (
 	FaaSInstance = kv.Key("faas.instance")
 )
 
+// Standard operation system process resource attribute keys.
```

Comment:
I think this is meant to be

```suggestion
// Standard operating system process resource attribute keys.
```

Reply:
whoops, thanks :+1: